README.md

```diff
@@ -1,3 +1,8 @@
+# LLVM-snippy
+
+This is LLVM project fork containing LLVM-snippy generator. Documentation of this
```

Comment:
This is an LLVM project fork containing the LLVM-snippy generator. Documentation for this tool is available [here].